tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{35}-django{111}-drf{39,latest},py{35,36}-django{20,21,22}-drf{39,latest}
+envlist = py35-django22-drf{39,latest}, py38-django{22,30}
```

Comment:
I think that we should remove the drf{39,  latest} part. And also the reference to drflatest in the .travis.yml file

Reply:
edx-platform is still on DRF 3.9.4, and this package is used there, so we need to keep testing for compatibility with that until it gets upgraded to a newer version.  So actually, we need to add back the `drf39` deps entry below and keep testing it in Travis.